File: FizzBuzz/FizzBuzz.rb
def FizzBuzz(number)	
	if number % 3 == 0 && number % 5 == 0
		"FizzBuzz"
	elsif number % 5 == 0
		"Buzz"
	elsif number % 3 == 0 
		"Fizz"
	else
		number			
	end
end

# for i in 1..100
# 	puts i.to_s 
# 	puts FizzBuzz(i)
# end

File: todo41/src/FizzBuzz.js
function FizzBuzz(NumNum){
	if(NumNum % 3 == 0)
		console.log("Fizz");
	if(NumNum % 5 == 0)
		console.log("Buzz");
}

for(var i = 1; i <= 100; i++){
	console.log(i) + FizzBuzz(i);
}
